.github/workflows/tests.yml

```diff
@@ -134,11 +134,17 @@ jobs:
   trial-olddeps:
     # Note: sqlite only; no postgres
     if: ${{ !cancelled() && !failure() }} # Allow previous steps to be skipped, but not fail
-    needs: linting-done
+    # needs: linting-done
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
 
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
+            override: true
```

Comment:
Is it worth pinning to a specific version of rustc here, to help keep builds more reproducible?